h2o-py/h2o/estimators/glm.py

```diff
@@ -2599,3 +2772,94 @@ def makeGLMModel(model, coefs, threshold=.5):
         m = H2OGeneralizedLinearEstimator()
         m._resolve_model(model_json["model_id"]["name"], model_json)
         return m
+
+    @staticmethod
+    def getConstraintsInfo(model):
```

Comment:
Please use `snake_case` not `cameCase` in python.

Reply:
But but Tomas, all the functions are in camelCase!!!! If I change this one, I need to change all of them in order to be consistent.....